src/GhSync/Services/Ado.cs

```diff
@@ -7,39 +7,39 @@
 
 namespace gh_sync;
 
-public static class Ado
+public class Ado : IAdo
 {
     internal const string ADOTokenName = "ado-token";
     internal const string ADOUriName = "ado-uri";
+    internal const string AdoProjectName = "ado-project";
 
-    internal static readonly string CollectionUri = Extensions.RetreiveOrPrompt(
+    internal static readonly string _CollectionUri = Extensions.RetreiveOrPrompt(
```

Comment:
Now that we have DI setup, this should likely be refactored into an IOptions pattern to make it easier to mix configuration from command-line arguments, from environment variables, or the registry (when available). Might be beyond the scope of this PR, but would be good as part of maturing the codebase.